AmplifyPlugins/Auth/AWSCognitoAuthPluginIntegrationTests/AWSAuthBaseTest.swift

```diff
@@ -18,21 +18,11 @@ class AWSAuthBaseTest: XCTestCase {
 
     let amplifyConfigurationFile = "testconfiguration/AWSCognitoAuthPluginIntegrationTests-amplifyconfiguration"
 
-    func initializeAmplifyWithError() throws {
+    func initializeAmplify() throws {
         let configuration = try TestConfigHelper.retrieveAmplifyConfiguration(
             forResource: amplifyConfigurationFile)
         let authPlugin = AWSCognitoAuthPlugin()
         try Amplify.add(plugin: authPlugin)
         try Amplify.configure(configuration)
     }
-
-    func initializeAmplify() {
```

Comment:
This was the original function and will now be superseded by the one that throws.